services/base-service/models/expert/api.js

```diff
@@ -34,63 +46,30 @@ function json_only(req, res, next) {
   }
 }
 
-async function sanitize(req, res, next) {
+function sanitize(req, res, next) {
```

Comment:
@rakunkel-ucd Here the sanitize middleware just wraps that call. basically.  It now longer uses a seperate authorization check, but instead uses the common `user_can_edit`